nrrd/tests/test_reading.py

```diff
@@ -130,7 +157,7 @@ def test_read_dup_field_error_and_warn(self):
             self.assertTrue("Duplicate header field: 'type'" in str(w[0].message))
 
             self.assertEqual(expected_header, header)
-            nrrd.reader._NRRD_ALLOW_DUPLICATE_FIELD = False
+            nrrd.reader.ALLOW_DUPLICATE_FIELD = False
 
```

Comment:
Shouldn't this be just `ALLOW_DUPLICATE_FIELD`?

Reply:
Yes, I **think** I fixed this in master, or maybe I fixed a different occurrence.